jr-stree/src/main/java/com/fasterxml/jackson/jr/stree/JacksonJrsTreeCodec.java

```diff
@@ -50,7 +51,16 @@ private JrsValue nodeFrom(JsonParser p) throws IOException
             return JrsBoolean.FALSE;
         case JsonTokenId.ID_NUMBER_INT:
         case JsonTokenId.ID_NUMBER_FLOAT:
-            return new JrsNumber(p.getNumberValue());
+            if (_useBigDouble) {
```

Comment:
what does bigDouble mean?

Reply:
changed the name to be more understandable